src/session/SessionMiddleware.ts

```diff
@@ -43,22 +41,21 @@ function sessionRequestHandler(config: CookieConfig, sessionStore: SessionStore)
 
             } catch (err) {
 
-                console.error(err);
+                loggerInstance().error(err);
                 response.clearCookie(config.cookieName);
                 delete request.session;
 
                 try {
                     const cookie = Cookie.createFrom(sessionCookie);
                     sessionStore.delete(cookie);
-                } catch (_) {
-                    console.error(_);
+                } catch (err) {
```

Comment:
Don't you get hint that `err` shadows parent `err` variable?

Reply:
no, but I now see why it should

Reply:
Not a big deal, it is safe to do so here.